sdks/java/core/src/main/java/org/apache/beam/sdk/util/ZipFiles.java

```diff
@@ -226,6 +228,8 @@ static void zipDirectory(
    * @throws IOException the zipping failed, e.g. because the input was not
    *     readable.
    */
+  @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
```

Comment:
I see. Yea, a comment about that would focus the suppression. Does this work, btw?

```
// We checked isDirectory() so this cannot actually return null
@SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
Iterable<File> files = sourceDirectory.listFiles();
```